src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/NativeCallableAttribute.cs

```diff
@@ -5,10 +5,16 @@
 namespace System.Runtime.InteropServices
 {
     /// <summary>
-    /// Any method marked with NativeCallableAttribute can be directly called from
+    /// Any method marked with <see cref="System.Runtime.InteropServices.NativeCallableAttribute" /> can be directly called from
     /// native code. The function token can be loaded to a local variable using LDFTN
```

Comment:
> The function token can be loaded to a local variable using LDFTN
 
This is not a good user-facing documentation. We should write this snippet as if the C# function pointers feature existed already.